content/03.results-and-discussion.md

```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
 
-Finally, the original EAGER tabular report format has been replaced with a much more extensive MultiQC (@doi:10.1093/bioinformatics/btw354) report. The original EAGER pipeline required users to look through many independent output directories and files to make full assessment of their sequencing data. Aggregation of all log files into a single interactive report will assist users in making fuller assessment of their sequencing and analysis runs. Most tools within nf-core/eager have a corresponding MultiQC module to ensure as complete evaluation as possible.
+Finally, the original EAGER tabular report format has been replaced with a much more extensive MultiQC (@doi:10.1093/bioinformatics/btw354) report. The original EAGER pipeline required users to look through many independent output directories and files to make full assessment of their sequencing data. Aggregation of all log files into a single interactive report will assist users in making fuller assessment of their sequencing and analysis runs. Most tools within nf-core/eager have a corresponding MultiQC module to ensure as complete an evaluation as possible.
 
 ## Accessibility
 
-Alongside the portable new pipeline report, we have written extensive documentation on all parts of running and interpreting the output of the pipeline. Given that a large fraction of aDNA researchers come from fields outside computational biology, such as social sciences - who often have limited computational training (such as J.A.F.Y), we have written documentation that also gives guidance on how to interpret each section of the report, specifically in the context of NGS and aDNA. This includes schematic images (by Z.F.) of best practices or expected output that published under CC-BY licenses to allow for use in other training material. We hope this open-access resource will make the aDNA discipline more accessible to researchers new to the field, by providing practical and 'applied' knowledge as to how aDNA characteristics translate to downstream analyses.
+Alongside the portable new pipeline report, we have written extensive documentation on all parts of running and interpreting the output of the pipeline. Given that a large fraction of aDNA researchers come from fields outside computational biology, and thus may have limited computational training, we have written documentation that also gives guidance on how to interpret each section of the report, specifically in the context of NGS and aDNA. This includes schematic images of best practices or expected output that published under CC-BY licenses to allow for use in other training material. We hope this open-access resource will make the aDNA discipline more accessible to researchers new to the field, by providing practical and 'applied' knowledge as to how aDNA characteristics translate to downstream analyses.
```

Comment:
I was trying to emphasis many aDNA researchers is not from any super-technical discpline (chemists or engineers will often have some comp. training) who get 0% computational training except for word and excel. Can you suggest a rephrase?

Reply:
Isn't my existing suggestions covering that?
> "Given that a large fraction of aDNA researchers come from fields outside computational biology, **and thus may have limited computational training**, we have written documentation [...]"